src/slc/outdated/tests/test_setup.py

```diff
@@ -1,26 +1,36 @@
+from plone import api
 from Products.CMFCore.utils import getToolByName
 from slc.outdated.tests.base import INTEGRATION_TESTING
+try:
+    from Products.CMFPlone.utils import get_installer
+except ImportError:
+    get_installer = None
 
-import unittest2 as unittest
+import unittest
 
 
 class TestInstall(unittest.TestCase):
     layer = INTEGRATION_TESTING
 
     def setUp(self):
         self.portal = self.layer['portal']
-        self.installer = getToolByName(self.portal, 'portal_quickinstaller')
+        if get_installer is not None:
+            self.installer = get_installer(self.portal, self.layer["request"])
+        else:
+            self.installer = getToolByName(self.portal, 'portal_quickinstaller')
 
     def test_product_installed(self):
-        self.failUnless(self.installer.isProductInstalled('slc.outdated'))
+        self.assertTrue(self.installer.isProductInstalled('slc.outdated'))
```

Comment:
`isProductInstalled` should be `is_product_installed` on Plone 5.2.
To make think easier I would just support Plone 5.2.

Reply:
I'll not look into that right now.